addons/addon-base-raas/packages/base-raas-services/lib/schema/register-data-source-bucket.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "accountId": {
+      "type": "string",
+      "pattern": "^[0-9]{12}$"
+    },
+    "name": {
+      "type": "string",
+      "minLength": 3,
+      "maxLength": 255
+    },
+    "region": {
+      "type": "string",
+      "maxLength": 100
+    },
+    "awsPartition": {
+      "type": "string",
+      "enum": ["aws", "aws-cn", "aws-us-gov"]
+    },
+    "kmsArn": {
+      "type": "string",
+      "maxLength": 90
+    },
```

Comment:
It can be different.  When an admin is registering a study, the admin is asked to specify if the bucket has a default kms and then for each study the admin is also asked if the study is using the bucket default kms or has its own kms key.  We capture enough information in both the bucket entity and the study entity to be able to determine if a study should be using the default bucket kms (when that is the case, the study kmsArn will be empty).  Let me know if this answered your question.